roles/lib_utils/action_plugins/master_check_paths_in_config.py

```diff
@@ -23,6 +23,7 @@
 
 ITEMS_TO_POP = (
     ('oauthConfig', 'identityProviders'),
+    ('auditConfig', 'auditFilePath'),
```

Comment:
This is not the appopriate place for this.  We should utilized ALLOWED_DIRS instead.